task/dev3/EnterUser/EnterUser.java

```diff
@@ -0,0 +1,29 @@
+package task.dev3.EnterUser;
+
+import task.dev3.User.*;
+
+import java.io.*;
+import java.util.*;
+
+/**
+ * Class set entered information.
+ */
+public class EnterUser {
+
+  /**
+   * Call other method for giv information about user and set it.
+   * @param user - user, which need to initialize.
+   */  
+  public void setEnterUserInfo(User user) {
+    Scanner in = new Scanner(System.in);
+    EnterUserName newName = new EnterUserName();
```

Comment:
if you need to read 1000k users you will have to make a lot of object instantiation